.github/scripts/toolchain/execute-cygwin-tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+CYGWIN_SOURCE_PATH=$SOURCE_PATH/cygwin
+CYGWIN_BUILD_PATH=$BUILD_PATH/cygwin
+CYGWIN_WINSUP_TEST_PATH=$BUILD_PATH/cygwin/$ARCH-$PLATFORM/winsup/testsuite
+
+echo "::group::Execute Cygwin tests"
+    pushd "$CYGWIN_WINSUP_TEST_PATH" || exit 1
+        make $CHECK_MAKE_OPTIONS check
```

Comment:
BUILD_MAKE_OPTIONS could be reused from config.sh

Reply:
I'd say it's better to have them separate. See https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/pull/301/files#diff-473314533d1b1ff05d40b29eafcb69b02f9b59ed8ed43a3d4fd9e68208c94aedR62 for an example.